docs/graphql/core/hasura-cli/hasura_seed_apply.rst

```diff
@@ -29,7 +29,7 @@ Examples
     hasura seed apply
 
     # Apply only a particular file:
-    hasura seed apply --file seeds/1234_add_some_seed_data.sql
+    hasura seed apply --file seed/1234_add_some_seed_data.sql
```

Comment:
@codeliezel have you actually tried this out? I don't know if the actual folder structure will change names.